src/main/groovy/com/google/protobuf/gradle/ProtobufExtract.groovy

```diff
@@ -110,7 +110,7 @@ class ProtobufExtract extends DefaultTask {
   protected void setDestDir(File destDir) {
     Preconditions.checkState(this.destDir == null, 'destDir already set')
     this.destDir = destDir
-    outputs.dir destDir
+    outputs.dirs destDir
```

Comment:
The difference between `dir()` and `dirs()` are the number of arguments they can take. Since you have only one arguments here, it doesn't seem this change is necessary.

Reply:
Oops. This is a typo (although functionally they have no difference).